achilles/app/app.go

```diff
@@ -304,6 +311,14 @@ func New(
 
 }
 
+func (app *App) setupUpgradeHandlers(cfg module.Configurator) {
+	app.UpgradeKeeper.SetUpgradeHandler(
+		v110.UpgradeName,
+		v110.CreateUpgradeHandler(cfg, app.sm),
+	)
+}
```

Comment:
You also need to add the new modules in storage. 
Docs: https://docs.cosmos.network/v0.50/learn/advanced/upgrade#add-storeupgrades-for-new-modules

Example with cosmos sdk 0.45
https://github.com/bze-alphateam/bze/blob/v6.0.0/app/app.go#L621